gui/qt/main_window.py

```diff
@@ -1710,6 +1738,11 @@ def pay_to_URI(self, URI):
 
 
     def do_clear(self):
+        self.crowdfunding_in_process= False
+        self.show_crowdsale_input_button.setVisible(False)
+        self.send_button.setVisible(True)
+        self.preview_button.setVisible(True) 
+        self.fee_e_label.setVisible(True)
```

Comment:
Need to properly undo "damage" (eg: changed send tab state) caused by crowdfund mode. This only partially undoes the changed state.